kolibri/core/templates/kolibri/base.html

```diff
@@ -6,8 +6,9 @@
   {% frontend_assets 'kolibri.core.KolibriCoreFrontEnd' %}
   {% base_frontend_async %}
   {% base_frontend_sync %}
-  <link type="text/css" rel="stylesheet" href="{% static 'core/css/components.css' %}" />
-  <link type="text/css" rel="stylesheet" href="{% static 'core/css/kolibri_style.css' %}" />
+	<!-- Testing the CSS bundler -->
+  <!-- <link type="text/css" rel="stylesheet" href="{% static 'core/css/components.css' %}" /> -->
+  <!-- <link type="text/css" rel="stylesheet" href="{% static 'core/css/kolibri_style.css' %}" /> -->
```

Comment:
It's preferable to just remove code instead of commenting it out. There is potential to create confusion for future maintainers since there is no context about why the code was taken out, if it was intentional, etc.